walkthroughs/week-4-libraries/maps/examples/hello-world/src/main/webapp/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <title>Hello Google Maps</title>
-  <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY"></script>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCLVdTakOHBo7xAtFRmiAO7NPlTLz_4iSg"></script>
```

Comment:
Did you restrict this API key?

https://developers.google.com/maps/documentation/javascript/get-api-key#restrict_key

Reply:
Yes, I restricted it.